draft-ietf-jose-hpke-encrypt.md

```diff
@@ -198,7 +198,7 @@ In HPKE JWE Integrated Encryption:
 - JWE Initialization Vector and JWE Authentication Tag MUST NOT be present.
 - JWE AAD MAY be present.
 - JWE Ciphertext is ciphertext as defined in Section 5.2 of {{RFC9180}}.
-- The HPKE info parameter MUST be set to an empty string.
+- The HPKE info parameter defaults to the empty string; mutually known private information MAY be used instead.
```

Comment:
```suggestion
- The HPKE info parameter defaults to the empty string; mutually known contextual information MAY be used instead.
```

Reply:
The use of "private" information is consistent with the contact KDF used by ECDH-ES which uses SuppPrivInfo (see https://csrc.nist.gov/csrc/media/projects/crypto-publication-review-project/documents/initial-comments/sp800-135r1-initial-public-comments-2023.pdf). 

SuppPrivInfo: An optional subfield, which could be null, that contains additional, mutually
known private information (e.g., a shared secret symmetric key that has been communicated
through a separate channel).

Reply:
The term "mutually known private information" is the one used in [NIST SP 800-56A](https://nvlpubs.nist.gov/nistpubs/SpecialPublications/NIST.SP.800-56Ar2.pdf) in the definition of the Concat KDF (which it calls "The Single-Step KDF").  Thus, I'm prone to keep it, rather than invent a new term.

Reply:
@selfissued @tireddy2 I would assume I could use any application-specific contextual values with the `info` value, right? HPKE (RFC9180) just says it is just application-supplied data which influences the generation of keys, e.g., KDF.

Reply:
@tireddy2 @selfissued is the `info` also part oft the AAD? I guess, I would like to understand if jose-hpke is going to support external contributors to the KDF (which I assume this PR is about) and external AAD?